server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaConfig.java

```diff
@@ -358,8 +360,8 @@ public ServerPort deserialize(JsonParser p, DeserializationContext ctx) throws I
                 return fail(ctx, root);
             }
 
-            final JsonNode proto = root.get("protocol");
-            if (proto == null || proto.getNodeType() != JsonNodeType.STRING) {
```

Comment:
How about retaining this for backward compatibility?

Reply:
Good point. Fixed!